deploy/deploy_app_server.yml

```diff
@@ -19,6 +19,7 @@
     - nodejs
     - redis
     - maven
+    - neo4j
```

Comment:
@olivermel Please remove this from the app_server.yml file, it's not needed there.